chunks/lang-ruby/install.sh

```diff
@@ -3,25 +3,10 @@ set -euo pipefail
 
 RUBY_VERSION=$1
 
-# the version of openssl changed in Jammy, Ruby 3.0 needs an older version
-# https://github.com/rvm/rvm/issues/5209#issuecomment-1114159447
-# another option: https://github.com/rvm/rvm/issues/5209#issuecomment-1134927685 or sudo apt install libssl-dev=1.1.1l-1ubuntu1.4
-if grep -q "3.0." <<<"${RUBY_VERSION}"; then
```

Comment:
no need for this anymore, as Ruby 3.0 is no longer supported and we no longer publish it